transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollChannel.java

```diff
@@ -327,17 +327,14 @@ final void epollInFinally(ChannelConfig config) {
             // See https://github.com/netty/netty/issues/2254
             if (!readPending && !config.isAutoRead()) {
                 clearEpollIn();
-            }
-        }
-
-        /**
-         * Will schedule a {@link #epollInReady()} call on the event loop if necessary.
-         * @param edgeTriggered {@code true} if the channel is using ET mode. {@code false} otherwise.
-         */
-        final void checkResetEpollIn(boolean edgeTriggered) {
```

Comment:
@normanmaurer - Please check the expected behavior of `checkResetEpollIn` should be preserved with these changes. The thought is we now have more fine grained knowledge if a read should be done and so we are using that instead of just reading if `edgeTriggered`.